### ISSUE TITLE
[WARNING] qtpylib.broker: Not submitting order

### ISSUE THREAD

**Issue by algotradingstrategies** [ranaroussi/qtpylib#73]
This sort of issue seems to have popped up in the past (Issue 63, 66, 71), reportedly fixed. But I still am getting it on the latest release. To be precise, I am getting it when submitting multiple orders in separate instruments immediately after each other. For instance, something like

```python
def on_tick(self, instrument):
    A = self.get_instrument('EURUSD_CASH') 
    B = self.get_instrument('GBPUSD_CASH')  
    A.buy_market(30000)
    B.buy_market(20000)
 ```
Then, the FIRST order (in A) is always submitted (and executed) correctly. The order in B is not. The first time, on_tick gets called, it seemingly does not submit any order - without error. On subsequent calls to on_tick, the submission in B throws the warning ```"[WARNING] qtpylib.broker: Not submitting order"``` (with reference to an order in B, which seems to be stuck).The order in A is submitted and executed perferctly always.

**Comment by ranaroussi** [comment]
This indeed keeps popping up... Looking into this

**Comment by ranaroussi** [comment]
Fixed in 1.5.8
